docker-compose.logging.yml

```diff
@@ -0,0 +1,85 @@
+version: '3'
+
+x-logging:
+  &logging
+  driver: loki
+  options:
+    loki-url: http://${PUBLIC_HOSTNAME}:3100/loki/api/v1/push
```

Comment:
Do you need the hostname or does the service name, i.e., `loki`, suffice?

Reply:
yes since the doman is resolved by the docker engine itself which is outside of the opentosca docker network